### DISCUSSION TITLE
Help implementing this .NET godot proposal

### DISCUSSION THREAD

**Discussion by niceWizzard** [godotengine/godot-proposals#6975] [Scripting]
I have made this proposal: #6944, I don't know how to implement it in C++ though.

**Comment by AThousandShips** [comment]
Please

You already have a proposal, ask the question there, or give it time

A proposal takes time to process, and the reviewing of the PR as well, and we're almost in feature freeze so it's very unlikely to end up in 4.1 at release, so it'll have to wait until it is released 

If you need support with this seek out the [other community channels](https://godotengine.org/community)
